src/bikeracks/templates/scripts.js

```diff
@@ -575,14 +593,17 @@ BikeMap.prototype.arrowHTML = function(state, currentUserId) {
 
 
 BikeMap.prototype.popupContent = function(state) {
-    let onlineStatus;
+    let onlineStatus,
+        voterStatus;
     if (this.auth.currentUser) {
         onlineStatus = this.auth.currentUser.uid;
     }
     else {
         onlineStatus = false
     }
     
+    voterStatus = onlineStatus === state.user_id;
```

Comment:
Where did the `state` come from? How could it have a different user's `user_id`?

Reply:
I think I was momentarily confused.